rtribbia/homework.txt

```diff
@@ -0,0 +1,86 @@
+1. a) "1" == 1
```

Comment:
Looks like you're pretty comfortable with problem #1! There are a few cases where edge cases can show up--the chief culprit here is `NaN`, which isn't equal to anything, including itself. That will, for example, be a case where 1q is `false`.  This also shows up in 1d, where `true` and `false` will also cause problems as booleans will autoconvert to strings when added to strings